jsonschema-core.md

```diff
@@ -2146,540 +2119,48 @@ a fragment in order to identify the specific schema object.
-      "evaluationPath": "/properties/bar/$ref/properties/bar-prop",
-      "schemaLocation": "https://json-schema.org/schemas/example#/$defs/bar/properties/bar-prop",
-      "instanceLocation": "/bar/bar-prop",
-      "errors": {
-        "minimum": "2 is less than or equal to 10"
-      }
-    }
-  ]
-}
-```
-
-```json "Passing results"
-{
-  "valid": true,
-  "details": [
-    {
-      "valid": true,
-      "evaluationPath": "",
-      "schemaLocation": "https://json-schema.org/schemas/example#",
-      "instanceLocation": "",
-      "annotations": {
-        "title": "root",
-        "properties": [
-          "foo",
-          "bar"
-        ]
-      }
-    },
-    {
-      "valid": true,
-      "evaluationPath": "/properties/foo/allOf/1",
-      "schemaLocation": "https://json-schema.org/schemas/example#/properties/foo/allOf/1",
-      "instanceLocation": "/foo",
-      "annotations": {
-        "title": "foo-title",
-        "properties": [
-          "foo-prop"
-        ],
-        "additionalProperties": [
-          "unspecified-prop"
-        ]
-      }
-    },
-    {
-      "valid": true,
-      "evaluationPath": "/properties/bar/$ref",
-      "schemaLocation": "https://json-schema.org/schemas/example#/$defs/bar",
-      "instanceLocation": "/bar",
-      "annotations": {
-        "title": "bar-title",
-        "properties": [
-          "bar-prop"
-        ]
-      }
-    },
-    {
-      "valid": true,
-      "evaluationPath": "/properties/foo/allOf/1/properties/foo-prop",
-      "schemaLocation": "https://json-schema.org/schemas/example#/properties/foo/allOf/1/properties/foo-prop",
-      "instanceLocation": "/foo/foo-prop",
-      "annotations": {
-        "title": "foo-prop-title"
-      }
-    },
-    {
-      "valid": true,
-      "evaluationPath": "/properties/bar/$ref/properties/bar-prop",
-      "schemaLocation": "https://json-schema.org/schemas/example#/$defs/bar/properties/bar-prop",
-      "instanceLocation": "/bar/bar-prop",
-      "annotations": {
-        "title": "bar-prop-title"
-      }
-    }
-  ]
-}
-```
+Errors are textual representations of individual validation failures, often
+intended for human consumers. This specification contains no requirements for
+the content of these errors.
 
-#### Hierarchical
+When represented in JSON, the key for this information MUST be "errors".
 
-The "Hierarchical" structure is a tree structure that follows the evaluation
-path during the validation process. Typically, it will resemble the schema as if
-all referenced schemas were inlined in place of their associated by-reference
-keywords.
+### Annotations
 
-All output units are included in this format.
+Many keywords are defined to produce annotations, whether intended for
+inter-keyword communication (e.g. between `properties` and
+`unevaluatedProperties`) or for application consumption (e.g. `title` or
+`readOnly`). Annotation values may be of any type and are defined by the
+keywords that produced them.
```

Comment:
It might be useful to have a link in here somewhere to the section that defines annotations.